support-frontend/app/controllers/Application.scala

```diff
@@ -66,8 +66,18 @@ class Application(
     RedirectWithEncodedQueryString(redirectUrl, request.queryString, status = FOUND)
   }
 
+  def supportGeoRedirect: Action[AnyContent] = GeoTargetedCachedAction() { implicit request =>
+    val redirectUrl = request.geoData.countryGroup match {
+      case Some(US) => buildCanonicalShowcaseLink("us")
+      case Some(Australia) => buildCanonicalShowcaseLink("au")
+      case _ => buildCanonicalShowcaseLink("uk")
```

Comment:
no need to repeat `buildCanonicalShowcaseLink` three times, just do it outside

Reply:
we are changing from a relative to an absolute url in the redirect too.  That is fine assuming it's correct, and I think it will be, but it is a subtle additional change.  https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location#Directives

Reply:
good point!